src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -201,27 +203,64 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
-                                        <Typography paragraph>
-                                          <Trans
-                                            i18nKey="registration.files_and_license.version_published_helper_text"
-                                            components={[<strong />]}
-                                          />
-                                        </Typography>
-                                        <Typography paragraph>
-                                          <Trans
-                                            i18nKey="registration.files_and_license.version_publishing_agreement_helper_text"
-                                            components={[<strong />]}
-                                          />
-                                        </Typography>
+                                        {registratorPublishesMetadataOnly ? (
+                                          <>
+                                            <Typography paragraph>
+                                              {t('registration.files_and_license.version_helper_text_metadata_only')}
+                                            </Typography>
+                                            <Typography paragraph>
+                                              <Trans
+                                                i18nKey="registration.files_and_license.version_accepted_helper_text_metadata_only"
+                                                components={[<strong />]}
+                                              />
+                                            </Typography>
+                                            <Typography paragraph>
+                                              <Trans
+                                                i18nKey="registration.files_and_license.version_published_helper_text_metadata_only"
+                                                components={[<strong />]}
+                                              />
+                                            </Typography>
+                                            <Typography paragraph>
+                                              <Trans
+                                                i18nKey="registration.files_and_license.version_publishing_agreement_helper_text_metadata_only"
+                                                components={[<strong />]}
+                                              />
+                                            </Typography>
+                                          </>
+                                        ) : (
+                                          <>
+                                            <Typography paragraph>
+                                              <Trans
+                                                i18nKey="registration.files_and_license.version_helper_text"
+                                                components={[<u />, <br />]}
```

Comment:
visste ikke at `<u>` var en ting en gang jeg 😅 
men sjekk warning her: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u

Reply:
kan ev. løses med å sende inn `<Box component="span" sx={{...}}>`

Reply:
Med mindre tanken er å ` mark text as having some form of non-textual annotation applied`, som jeg ikke helt forstår hva betyr 😅